extension/Config.jsm

```diff
@@ -0,0 +1,69 @@
+/* to use:
+
+- Recall this file has chrome privileges
+- Cu.import in this file will work for any 'general firefox things' (Services,etc)
+  but NOT for addon-specific libs
+*/
+
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(config|EXPORTED_SYMBOLS)" }]*/
+const EXPORTED_SYMBOLS = ["config"];
+
+// var slug = "shield-example-addon"; // should match chrome.manifest;
+
+const config = {
+  study: {
+    studyName: "share-button-study", // no spaces, for all the reasons
+    variation: {
+      name: "ALL",
+    }, // optional, use to override/decide
+    weightedVariations: [
+      { name: "control", weight: 1 },
+      { name: "DOORHANGER", weight: 2 },
+      { name: "HIGHLIGHT", weight: 0.5 },
+    ],
+    /** **endings**
+      * - keys indicate the 'endStudy' even that opens these.
+      * - urls should be static (data) or external, because they have to
+      *   survive uninstall
+      * - If there is no key for an endStudy reason, no url will open.
+      * - usually surveys, orientations, explanations
+      */
+    endings: {
+      /** standard endings */
+      "user-disable": {
+        baseUrl: "data:,You uninstalled",
+      },
+      ineligible: {
+        baseUrl: "http://www.example.com/?reason=ineligible",
```

Comment:
We'll need to get final urls for this, but I can imagine not having them yet. Maybe use data:uris instead? Or if you leave this out entirely will it skip opening a new tab?